### DISCUSSION TITLE
Move to next window (include virtual screens)

### DISCUSSION THREAD

**Discussion by skyf5** [rxhanson/RectanglePro-Community#45] [Ideas]
Can you add the option or consider move to window, to include virtual screens? i.e. ^ + arrow (left or right)

**Comment by rxhanson** [comment]
Apple doesn’t provide a supported way for moving a window among spaces (desktops). Other apps that allow this behavior are using hacks that might not work for all users, and might break in future versions of macOS. I would definitely add this functionality in if Apple provided a supported API for it.

**Comment by rxhanson** [comment]
In the most recent release, 3.0.12, I added in "move to next/prev space" keyboard shortcuts.
